### PR TITLE
nrf_dm: Adding version 0.7.1 of nrf_dm

### PR DESCRIPTION
**Pull request by eriksandgren**
Add a new library version that features some minor improvements
and two new functions  in the nrf_dm.h header

### CODE REVIEW

File: nrf_dm/doc/CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,21 @@ Changelog
...
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+* :c:func:`void nrf_dm_get_hopping_sequence(const nrf_dm_config_t * config, uint8_t hopping_sequence[NRF_DM_CHANNEL_MAP_LEN])`, to get the channel hopping sequence used during the measurement procedure.
+* :c:func:`uint32_t nrf_dm_get_duration_us(const nrf_dm_config_t * config)`, to get the duration of a measurement procedure in microseconds.
+
+Changes
+=======
+* Invalid results of :c:func:`nrf_dm_high_precision_calc` now returns `NaN`.
```
Comment (wiba-nordic) [inline nrf_dm/doc/CHANGELOG.rst:24]:
```suggestion
* Invalid results of :c:func:`nrf_dm_high_precision_calc` now return `NaN`.
```

File: nrf_dm/doc/CHANGELOG.rst [hunk 2]
```diff
@@ -9,6 +9,21 @@ Changelog
...
+* Increase the number of measurements with quality :c:enumerator:`NRF_DM_QUALITY_OK`.
```
Comment (wiba-nordic) [inline nrf_dm/doc/CHANGELOG.rst:25]:
```suggestion
* The number of measurements can now be increases with the :c:enumerator:`NRF_DM_QUALITY_OK` quality.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: nrf_dm/include/nrf_dm_version.h [hunk 3]
```diff
@@ -1,15 +1,47 @@
 /*
- * Copyright (c) 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2020 - 2022, Nordic Semiconductor ASA
+ * All rights reserved.
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * Redistribution and use in source and binary forms, with or without modification,
```
Comment (Tschet1) [inline nrf_dm/include/nrf_dm_version.h:5]: I think this change should be undone.

File: nrf_dm/include/nrf_dm.h [hunk 4]
```diff
@@ -249,19 +249,21 @@ typedef struct {
 #define NRF_DM_DEFAULT_TX_POWER RADIO_TXPOWER_TXPOWER_0dBm
 #endif
 
+#define NRF_DM_DEFAULT_ANTENNA_COMP (NRF_DM_ANTENNA_COMP_1_1)
 /**
  * @brief Default configuration define, note that both role and access_address
  * needs to be configured even when using the default config.
  */
-#define NRF_DM_DEFAULT_CONFIG                                                              \
-	((nrf_dm_config_t){                                                                    \
```
Comment (Tschet1) [inline nrf_dm/include/nrf_dm.h:257]: what changed here? Is using whitespaces instead of tabs?

File: nrf_dm/doc/CHANGELOG.rst (hunk 2 above)
Comment (Tschet1) [reply nrf_dm/doc/CHANGELOG.rst:25]: It's more like:
The number of measurements with quality c:enumerator:`NRF_DM_QUALITY_OK` was increased.

**Review by Tschet1 — CHANGES_REQUESTED** [review]

File: nrf_dm/include/nrf_dm.h (hunk 4 above)
Comment (eriksandgren) [reply nrf_dm/include/nrf_dm.h:257]: Spaces on both. Just aligned the backslashes

File: nrf_dm/include/nrf_dm_version.h (hunk 3 above)
Comment (eriksandgren) [reply nrf_dm/include/nrf_dm_version.h]: Good point, missed this part.

File: nrf_dm/doc/CHANGELOG.rst (hunk 2 above)
Comment (eriksandgren) [reply nrf_dm/doc/CHANGELOG.rst]: Updated to @Tschet1  suggestion here

File: nrf_dm/doc/CHANGELOG.rst (hunk 1 above)
Comment (eriksandgren) [reply nrf_dm/doc/CHANGELOG.rst]: Done.

**Review by wiba-nordic — APPROVED** [review]
Changelog looks good now.

**Review by Tschet1 — APPROVED** [review]

File: nrf_dm/doc/CHANGELOG.rst [hunk 5]
```diff
@@ -9,6 +9,21 @@ Changelog
...
 All the notable changes to this project are documented in this file.
 
+main
+****
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+* :c:func:`void nrf_dm_get_hopping_sequence(const nrf_dm_config_t * config, uint8_t hopping_sequence[NRF_DM_CHANNEL_MAP_LEN])`, to get the channel hopping sequence used during the measurement procedure.
```
Comment (Tschet1) [inline nrf_dm/doc/CHANGELOG.rst:19]: I think the build fails because of this.
Probably :c:func:`nrf_dm_get_hopping_sequence` should be used.

Comment (Tschet1) [reply nrf_dm/doc/CHANGELOG.rst:19]: https://github.com/nrfconnect/sdk-nrf/runs/8073180130?check_suite_focus=true

Comment (wiba-nordic) [reply nrf_dm/doc/CHANGELOG.rst:19]: Agreed.

Comment (eriksandgren) [reply nrf_dm/doc/CHANGELOG.rst]: Thanks, giving it another try

**Comment by eriksandgren** [comment]
Verification in https://github.com/nrfconnect/sdk-nrf/pull/8561

**Pull request merged**